content/zh/search.html

```diff
@@ -0,0 +1,155 @@
++++
+title = "检索"
+#menu = "main"
+#weight = -80
++++
+
+<style>
+  /* 手机适配 */
+  @media screen and (max-width: 500px) {
+     .search{
+       padding-right: 25px;
+     }
+
+     .search input{
+       width: 100%;
+     }
+
+     .search button{
+       display: none;
+     }
+  }
+  /* 电脑适配 */
+  @media screen and (min-width: 500px) {
+      .search{
+        width: 500px;
+      }
+
+      .search input{
+        width: 444px;
+      }
+  }
+
+  /* 通用样式 */
+  .search{
+    margin: auto;
+  }
+
+  .search input{
+    outline: none;
+    border: 2px solid #c05b4d;
+    height: 32px;
+    padding: 10px;
+  }
+  .search button{
+    outline: none;
+    border: 0px;
+    height: 32px;
+    width:42px;
+    position:absolute;
+    background-color:#c05b4d ;
+  }
+  .search .icon{
+    width: 28px;
+    height: 28px;
+  }
+</style>
+<div class="search" οnlοad="getTestXmlValue();">
+  <input type="text" placeholder="请输入检索的内容..." id="search-key" />
```

Comment:
We can use the i8n to avoid host the same code twice.

Reply:
Due to the problem of code running mechanism, the retrieval function cannot read the I18N file.